wagtailmenus/models/menuitems.py

```diff
@@ -126,6 +127,25 @@ def clean(self, *args, **kwargs):
             raise ValidationError({'link_text': msg})
         super().clean(*args, **kwargs)
 
+    def get_active_class_for_request(self, request=None):
+        # Returns the 'active_class' for a custom link item.
+        if app_settings.CUSTOM_URL_SMART_ACTIVE_CLASSES:
+            # new behaviour
+            parsed_url = urlparse(self.link_url)
+            if not parsed_url.netloc:
+                # do nothing if the custom link URL has a domain
```

Comment:
I think I would prefer to do...

```
if parsed_url.netloc:
    return ''
```

Here and exit early. We can then lose the indentation on the two conditionals below it. 

We can also safely change all of the `elif` statements in the method to `if` to make this extra readable :)